aoe4/formatters.js

```diff
@@ -109,7 +109,12 @@ class NightbotDefaultFormatter {
       const teams = match.teams.map(t => this.formatMatchTeam(match, t, true)).join(' vs ');
       msg = `Custom on ${match.map} between ${teams}`;
     }
-    msg += `, ${formatAge(match.started_at)} ago`;
+    if (match.ongoing) {
+        msg += " [started "
```

Comment:
I think it's worth it duplicate the formatAge part, so
> msg += ` [started ${formatAge(...)} ago]`;
> else
> msg += ` [played ${formatAge(...)} ago]`;

and same for played.
It makes it more readable and adjustable in the future. coz the `[` `]` delimiters are on one line.